crt_portal/cts_forms/templates/landing.html

```diff
@@ -294,14 +301,21 @@ <h2 class="h3__display text__reverse">{% trans "We determine next steps and get
       <div class="grid-row grid-gap">
         <div class="tablet:grid-col-8">
           <h2 class="h2__display">{% trans "Have you or someone you know experienced a civil rights violation?" %}</h2>
+          {% if shutdown_mode == True %}
+          <p>We are currently not accepting reports. See the page banner for more information.</p>
+          <a class="usa-button usa-button--big crt-button--large" aria-disabled="true">{% trans "Start a report" %}</a>
+          {% else %}
           <a class="usa-button usa-button--big crt-button--large" href="{% url 'crt_report_form' %}">{% trans "Start a report" %}</a>
+          {% endif %}
         </div>
       </div>
+      {% if shutdown_mode == False %}
```

Comment:
nit: prefer plain language

```suggestion
      {% if not shutdown_mode %}
```